docs/application/native/guides/text-input/tts-engine.md

```diff
@@ -0,0 +1,369 @@
+# Text-to-speech Engine
+
+
+Text-to-speech Engine (TTSE) is an engine to synthesize voice from input texts and to send the synthesized sound data to TTS framework. 
+
+TTSE developers can provide TTSE service users, to apply TTSE with functions that is necessary to operate the engine.
```

Comment:
This sentence sounds unclear to me, It would be better if you could reframe it.
For example:
**_TTSE developers can provide TTSE service users with the functions that are necessary to operate the engine._**

Reply:
Thank you for your suggestion.
I changed that sentence with your suggestion. Please check and review it one more time.